src/test/qgstest.h

```diff
@@ -26,8 +26,10 @@
   QT_END_NAMESPACE \
   int main(int argc, char *argv[]) \
   { \
+    QTemporaryDir dir; \
     QgsApplication app(argc, argv, false); \
-    app.init(); \
+    app.init( dir.path() ); \
+    dir.setAutoRemove( false ); \
```

Comment:
Why not autoremoving it ? Shall it be determined by some env variable ?

Reply:
Good point, I didn't realize that the scope is the complete main 
```suggestion
```